diff --git a/recipes/harfbuzz.recipe b/recipes/harfbuzz.recipe
index 3e4b9018..b896327f 100644
--- a/recipes/harfbuzz.recipe
+++ b/recipes/harfbuzz.recipe
@@ -13,6 +13,7 @@ class Recipe(recipe.Recipe):
     deps = ['fontconfig', 'cairo', 'glib']
     meson_options = {'icu': 'disabled',
                      'glib': 'enabled',
+                     'docs': 'disabled',
                      'cairo': 'enabled', # Is this needed? Is it only used for cmd line utils?
                      'gobject': 'disabled',
                      'icu_builtin': 'false',
