diff --git a/client/iOS/CMakeLists.txt b/client/iOS/CMakeLists.txt
index 4970cee08..f3f2029b2 100644
--- a/client/iOS/CMakeLists.txt
+++ b/client/iOS/CMakeLists.txt
@@ -130,3 +130,14 @@ set(${MODULE_PREFIX}_LIBS ${${MODULE_PREFIX}_LIBS} winpr freerdp)
 target_link_libraries(${MODULE_NAME} ${${MODULE_PREFIX}_LIBS})
 
 set_property(TARGET ${MODULE_NAME} PROPERTY FOLDER "Client/iOS")
+
+add_library(${MODULE_NAME}Lib ${${MODULE_PREFIX}_SRCS}
+        ${IOS_CLIENT_ADDITIONS_SRCS} ${IOS_CLIENT_ADDITIONS_HDRS}
+        ${IOS_CLIENT_CONTROLLERS_SRCS} ${IOS_CLIENT_CONTROLLERS_HDRS}
+        ${IOS_CLIENT_FREERDP_SRCS} ${IOS_CLIENT_FREERDP_HDRS}
+        ${IOS_CLIENT_MISC_SRCS} ${IOS_CLIENT_MISC_HDRS}
+        ${IOS_CLIENT_MODELS_SRCS} ${IOS_CLIENT_MODELS_HDRS}
+        ${IOS_CLIENT_VIEWS_SRCS} ${IOS_CLIENT_VIEWS_HDRS}
+        ${${MODULE_NAME}_RESOURCES})
+
+target_link_libraries(${MODULE_NAME}Lib ${${MODULE_PREFIX}_LIBS})
